Orange/classification/tree.py

```diff
@@ -64,6 +64,8 @@ def __init__(
         self.min_samples_split = min_samples_split
         self.sufficient_majority = sufficient_majority
         self.max_depth = max_depth
+        self.params = {k: v for k, v in vars().items()
+                       if k not in ('args', 'kwargs')}
```

Comment:
I still don't understand what is the content of `vars`. It contains `binarize`, `max_depth`, `min_samples_leaf`, `min_samples_splet` and `sufficient_majority`; if these need to be copied to `params`, I'd copy them explicitly. Besides, it probably contains `self`, and besides that, all global names in the module, which you don't want to copy.

I am probably missing something here.